packages/main/src/commands/actions/add.ts

```diff
@@ -9,6 +9,8 @@ import { askVerifyEmailFlow } from '../../utils/askVerifyEmailFlow'
 import { askVerifySmsFlow } from '../../utils/askVerifySmsFlow'
 import withStandardErrors from '../../utils/errorHandling'
 
+const GDRIVE_PDF_UPLOADER_URL = 'gdrive.url' // FIXME what's the url??
```

Comment:
`https://us-central1-mailscript-firebase.cloudfunctions.net/` + path/to/function

Reply:
Done